training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ! aws ecr get-login-password --region {region} | docker login --username AWS --password-stdin 763104351884.dkr.ecr.{region}.amazonaws.com
</pre><p>User's AWS account ID rather than 763104351884?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>No this login is needed to fetch the base DLC image. Login to customer account is done as part of build_and_push script.</p>